src/ec/packages/ec-component-expandable/expandable.html.twig

```diff
@@ -0,0 +1,102 @@
+{#
+  Parameters:
+    - "id" (string) (default: '')
+    - "label_expanded" (string) (default: '')
+    - "label_collapsed" (string) (default: '')
+    - "label_collapsed" (string) (default: '')
+    - "expanded" (string) (default: '')
```

Comment:
there are other component where we handle boolean directly instead of strings (https://github.com/ec-europa/ecl-twig/blob/master/src/ec/packages/ec-component-language-list/language-list.html.twig for instance), and it seems to wroks correctly

Reply:
Found a way to implement booleans, I hope it's better.